### ISSUE TITLE
AUTOREV in zeroc recipes?

### ISSUE THREAD

**Issue by ssfivy** [zeroc-ice/meta-zeroc#2]
Hello,

I noticed that the source revision uses AUTOREV instead of fixed revisions on these files:

- zeroc-ice_3.7.3.bb
- zeroc-ice_3.7.4.bb

Is this intentional? I would expect recipes with fixed version numbers to contained fixed git revision. Both zeroc-ice_3.7.0.bb and zeroc-ice_3.7.1.bb seem to contain fixed revisions too.

**Comment by externl** [comment]
Yes, you're right. I think we unintentionally copied `AUTOREV` from our `zeroc-ice_3.7.bb` formula after releasing. These fixed releases should use the commit hash.

**Comment by ssfivy** [comment]
Okay, so only the zeroc-ice_3.7.bb file is supposed to use AUTOREV for those who always want the latest version. Also in this file I noticed:

PV = "3.7.3+git${SRCPV}"

should this be:

PV = "3.7+git${SRCPV}"

?

**Comment by externl** [comment]
I think the issue is with version priority, but I don't quite remember.  Since Ice 3.7.4 is out it should at least be:
`PV = "3.7.4+git${SRCPV}"`

@pepone  do you remember?

**Comment by pepone** [comment]
> I think the issue is with version priority, but I don't quite remember. Since Ice 3.7.4 is out it should at least be:
> `PV = "3.7.4+git${SRCPV}"`
> 
> @pepone do you remember?

`PV = "3.7.4+git${SRCPV}"` sounds right to me
